src/System.Text.Json/src/System/Text/Json/Serialization/JsonClassInfo.AddProperty.cs

```diff
@@ -182,12 +190,12 @@ private JsonPropertyInfo AddProperty(Type propertyType, PropertyInfo propertyInf
 
         internal JsonPropertyInfo CreateRootObject(JsonSerializerOptions options)
         {
-            return CreateProperty(Type, Type, Type, null, Type, options);
+            return CreateProperty(Type, Type, Type, null, Type, null, options);
```

Comment:
nit: we should use named arguments here and below.

Reply:
I don't think this was addressed.